api/specs/web-server/_projects_crud.py

```diff
@@ -45,28 +46,32 @@
 )
 
 
-@router.post(
-    "/projects",
-    response_model=Envelope[TaskGet],
-    summary="Creates a new project or copies an existing one",
-    status_code=status.HTTP_201_CREATED,
-)
-async def create_project(
-    _params: Annotated[ProjectCreateParams, Depends()],
-    _create: ProjectCreateNew | ProjectCopyOverride,
-    x_simcore_user_agent: Annotated[str | None, Header()] = "undefined",
+class _ProjectCreateHeaderParams(BaseModel):
+    x_simcore_user_agent: Annotated[str | None, Header()] = "undefined"
```

Comment:
Q: why did you remove the description here?

Reply:
I did not, it was not there. I will take it from